### PR TITLE
Add itk-jupyter-widgets for 3D visualisations

### PR DESCRIPTION
**Pull request by kaedonkers**
For use in Hero-Demo
https://github.com/InsightSoftwareConsortium/itk-jupyter-widgets

### CODE REVIEW

**Comment by kaedonkers** [comment]
Just double-checking: Will this be available in Pangeo `dev` rather than a tagged release?

**Comment by jacobtomlinson** [comment]
Once merged and built it will be available as the `dev` image choice (second one down) from either pangeo stack. Be aware that the dev image isn't pre-pulled onto hosts so start times can  be very slow and occasionally time out.

image

File: Dockerfile [hunk 1]
```diff
@@ -55,6 +55,7 @@ RUN conda install --yes \
     plotly \
     pyviz_comms>=0.7.0 \
     voila \
+    itkwidgets \
```
Comment (jacobtomlinson) [inline Dockerfile]: Please move up in this list. It's alphabetical.

File: Dockerfile [hunk 2]
```diff
@@ -71,7 +72,9 @@ RUN jupyter labextension install \
     @jupyterlab/plotly-extension \
     @jupyterlab/statusbar \
     @jupyter-widgets/jupyterlab-sidecar \
+    @jupyter-widgets/jupyterlab-manager \
```
Comment (jacobtomlinson) [inline Dockerfile]: This list doesn't seem to be alphabetical, but it probably should be.

I would be tempted to suggest that we have all the `@` starting ones alphabetical followed by the rest alphabetical.

Comment (kaedonkers) [reply Dockerfile]: :+1:

**Comment by kaedonkers** [comment]
@jacobtomlinson they gone been alphabetised

**Pull request merged**